### PR TITLE
Feat: Cloud Gateways Documentation

### PR DESCRIPTION
**Pull request by Guaris**
Cloud Gateways: 
https://konghq.atlassian.net/browse/DOCU-3686

[Preview](https://deploy-preview-7097--kongdocs.netlify.app/konnect/)
**Ready for review**
[Overview Page](https://deploy-preview-7097--kongdocs.netlify.app/konnect/gateway-manager/dedicated-cloud-gateways/)
[Networking and Peering information](https://deploy-preview-7097--kongdocs.netlify.app/konnect/network-resiliency/#how-does-network-peering-work-with-dedicated-cloud-gateway-nodes)
[Supported regions](https://deploy-preview-7097--kongdocs.netlify.app/konnect/geo/#dedicated-cloud-gateways)
[Transit Gateways](https://deploy-preview-7097--kongdocs.netlify.app/konnect/gateway-manager/data-plane-nodes/transit-gateways/)
[How to upgrade data planes](https://deploy-preview-7097--kongdocs.netlify.app/konnect/gateway-manager/data-plane-nodes/upgrade/)
[Custom Domains](https://deploy-preview-7097--kongdocs.netlify.app/konnect/reference/custom-dns/)
[How to use CGW](https://deploy-preview-7097--kongdocs.netlify.app/konnect/gateway-manager/provision-cloud-gateway/)
[Diagrams]()
[API Spec]()

-----
- [x] [Overview Page + change log](https://github.com/Kong/docs.konghq.com/pull/7084) 
- [x] [Terminology updates](https://github.com/Kong/docs.konghq.com/pull/7093)
- [x] [Networking and Peering Information](https://github.com/Kong/docs.konghq.com/pull/7092)
- [x] [New Supported AWS regions](https://github.com/Kong/docs.konghq.com/pull/7110)
- [x] [Transit gateways](https://github.com/Kong/docs.konghq.com/pull/7151)
- [x] [Updates to getting started docs](https://github.com/Kong/docs.konghq.com/pull/7151)
- [x] [How to upgrade data plane nodes](https://github.com/Kong/docs.konghq.com/pull/7155)
- [x] [CGW Operations](https://github.com/Kong/docs.konghq.com/pull/7156)
- [x] [Custom DNS](https://github.com/Kong/docs.konghq.com/pull/7163)
- [x] Diagrams
- [x] [API Spec](https://github.com/Kong/docs.konghq.com/pull/7207)

Preview Link: https://deploy-preview-7097--kongdocs.netlify.app/konnect/

### CODE REVIEW

**Comment by subnetmarco** [comment]
- [ ] The image in the Overview page is oversized.
- [ ] In the regions we should also add the geographical name
- [ ] In transit gateway we should add an example and perhaps a short 60 seconds video that shows how to do it.
- [ ] In how to upgrade we should have an image that shows where to click.
- [ ] In networking and peering we need architectural diagrams that explain the setup like:

image

File: app/konnect/gateway-manager/dedicated-cloud-gateways.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+Dedicated Cloud Gateways are data plane nodes that are fully managed by Kong in {{site.konnect_short_name}}.
+	
+You don't need to host any data planes, while maintaining control over the size and location of the gateway infrastructure. This allows Kong to autoscale your nodes for you and reduces your operational complexity.
+
+
+Dedicated Cloud Gateways offer the following benefits:
+* {{site.konnect_short_name}} handles gateway upgrades for you
+* A public or private mode to decide who can view your APIs. In public mode, powered by Kong's public Edge DNS for clusters, you can expose your APIs to the internet. 
+* Auto-pilot mode
```
Comment (acgoldsmith) [inline app/konnect/gateway-manager/dedicated-cloud-gateways.md:14]:
```suggestion
* Automatic scaling of data plane nodes to meet changing demand with Autopilot mode
```

**Review by acgoldsmith — APPROVED** [review]
LGTM!!!! 

One tiny note against the overview page, repeating what autopilot does from further down the page. 

Congrats on this project and on delivering as a team, @Guaris , @cloudjumpercat , @lena-larionova !

**Comment by rosskukulinski** [comment]
Thanks team for pulling this together! I know it came in hot and without nearly enough detail 😅 

Feel free to merge in the morning before 9am ET / 6am PT.

**Pull request merged**